src/components/forums/MyForums.js

```diff
@@ -227,6 +232,13 @@ const MyForums = () => {
         setForumDescErrorText('');
     }
 
+    const [forumTag, setForumTag] = React.useState('');
+    const forumTagList = ["School", "Co-op", "Funny", "Debate", "Rant", "Interview", "Class Review", "Good News"];
```

Comment:
"other" option

Reply:
Please refer to my comment above, lmk what you think thanks!